3p/recaptcha.js

```diff
@@ -102,16 +108,16 @@ window.initRecaptcha = function() {
 
 /**
  * Function to initialize our IframeMessagingClient
- * @param {Window} window
+ * @param {Window} win
  * @param {*} grecaptcha
  * @param {Object} dataObject
  */
-function initializeIframeMessagingClient(window, grecaptcha, dataObject) {
-  iframeMessagingClient = new IframeMessagingClient(window);
+function initializeIframeMessagingClient(win, grecaptcha, dataObject) {
```

Comment:
where does `dataObject` come from? i want to make sure this doesn't need to be a `JsonObject` type instead

Reply:
It comes from the JSON placed in the ifram name. I'll set as JSONObject, because you are probably right 😄